pyvisa-py/protocols/usbutil.py

```diff
@@ -205,7 +205,11 @@ def cm(dev):
             if custom_match is not None and not custom_match(dev):
                 return False
             for attr, pattern in attrs.items():
-                if not fnmatch(getattr(dev, attr).lower(), pattern.lower()):
+                try:
+                    value = getattr(dev, attr)
+                except Exception:
```

Comment:
Could we use a more specific error here since we know how this could fail ? (and if it fails for a different reason it may be nice to know about it)

Reply:
OK, changed it to be consistent with the check in usb.py.